content/en/tracing/trace_collection/single-step-apm.md

```diff
@@ -59,30 +62,390 @@ For example, for a Docker Linux container:
      -v /var/run/docker.sock:/var/run/docker.sock:ro \
      gcr.io/datadoghq/agent:7
    ```
-   <div class="alert alert-info">You can optionally set an environment for your services and other telemetry that pass through the Agent. Read <a href="#tagging-observability-data-by-environment">tagging observability data by environment</a> to learn how. </div>
+   Replace `<YOUR_DD_API_KEY>` with your [Datadog API][4].
+
 3. Restart the Docker containers.
 4. [Explore the performance observability of your services in Datadog][2].
 
+   <div class="alert alert-info">You can optionally set an environment for your services and other telemetry that pass through the Agent. Read <a href="#tagging-observability-data-by-environment">tagging observability data by environment</a> to learn how. </div>
+
 [1]: /agent/remote_config
 [2]: /tracing/service_catalog/
 
 {{% /tab %}}
 
+{{% tab "Kubernetes" %}}
+
+You can enable APM when installing the Agent with the [Datadog Operator](#operator) or [Helm chart](#helm-chart).
+
+### Operator
+
+The Datadog Operator is a way to deploy the Datadog Agent on Kubernetes and OpenShift.
+
+#### Prerequisites
+
+- Kubernetes Cluster version v1.20.X+: Tests were done on v1.20.0+; should be supported in v1.11.0+. For earlier versions, because of limited CRD support, the Operator may not work as expected.
```

Comment:
```suggestion
- Kubernetes Cluster version v1.20.X+ and v1.11.0+. For earlier versions, because of limited CRD support, the Operator may not work as expected.
```
This construction is a little confusing and I think we should avoid "should". Also, I wonder whether the versions that were tested need to be included? If we say a range is supported, the user can probably infer that we've tested that range.